plutarch-test/src/Plutarch/Test/Golden.hs

```diff
@@ -0,0 +1,184 @@
+{-# LANGUAGE ImpredicativeTypes #-}
```

Comment:
If you don't want to support GHC 8.10, then why not make the instance for `ClosedTerm`?

Reply:
Well, `plutarch-test` the library should be supported in 8.10 -- we are using this in our smart contract.

Reply:
Well then you should probably remove this.